libc/string/string_tokpbrkstr.c

```diff
@@ -729,9 +738,10 @@ TEST(string_str, empty_args)
 {
 	const char str[] = "Lorem";
 
-
 	char *asciiStr = testdata_createCharStr(ASCII_LENGTH);
 
+	TEST_ASSERT_NOT_NULL(asciiStr);
```

Comment:
Thanks for adding also those missing checks, Could you please also do it next to all `testdata_createCharStr` calls?